rpc_jobs/irr_role_test.yml

```diff
@@ -34,8 +37,18 @@
         context: liberty
       - image: xenial
         context: kilo
+      - scenario: ceph
```

Comment:
This would just be for master, we aren't deploying rpc-o + ceph, so the kilo/liberty/mitaka bits are meaningless.

Reply:
@andymcc Yeah, this is excluding all of those branches so that it only uses the ``ceph`` scenario for the master branch.